be/src/storage/lake/starlet_location_provider.cpp

```diff
@@ -30,5 +30,20 @@ std::string StarletLocationProvider::root_location(int64_t tablet_id) const {
     return build_starlet_uri(tablet_id, "");
 }
 
+StatusOr<std::string> StarletLocationProvider::real_location(std::string virtual_path) const {
```

Comment:
think of if this ability can be provided by the underlying FileSystem, so it can be as simple as 
```
    string real_path = fs::GetFileSystem(virtual_path)->abs_path(virtual_path);
```

Reply:
This is also an option. However, my only concern is that the path returned by fs through `abs_path()` cannot be processed by fs itself. Is it a bit strange?

Reply:
I see, it can be processed by the fs from fslib::FileSystem, but can't be processed by StarletFileSystem. that's a little bit awkward.